### ISSUE TITLE
too complex

### ISSUE THREAD

**Issue by gatspy** [tweekmonster/dootfiles#1]
Your dotfiles really complex , after installation , nvim run error in customer_tomorrow.vim `Unknown function: colorpal # begin CPHL not command`
 in zplug install script the url should be change.

**Comment by tweekmonster** [comment]
Oh, don't install it.  It's very specific to my setup.  I leave it online as a reference only.  `CPHL` refers to a plugin that's under development: https://github.com/tweekmonster/colorpal.vim

**Comment by tweekmonster** [comment]
If you ran `make install` or `./install`, you should run `make clean-links` to remove the symlinks it created.

**Comment by gatspy** [comment]
you are so cool! your dotfiles so profound, i cannot make heads or tails of it.
Why not consider creating a awesome lightweight dotfiles repo.

**Comment by tweekmonster** [comment]
Haha, thanks :)

I tried making my dot files simple, but over time they will eventually become one large and unreadable config file.  That became frustrating when I wanted to change variables.

I think you're mainly interested in the Vim config, so here's a brief explanation of how it works so it's a little bit easier to understand it:
- In [init.vim](https://github.com/tweekmonster/dootfiles/blob/master/vim/init.vim), it sets expected directories, then loads configs for `all`, `vim`, or `nvim`.
- In [_source](https://github.com/tweekmonster/dootfiles/tree/master/vim/_source), scripts are named by their purpose and have a number prefixed for the load priority.  The number prefix is similar to how Debian names scripts in `/etc/rc*.d`.
- Scripts that that start `01-` or `02-` are always loaded.
- Scripts that start with `03-` and up requires the plugins directory to exist.  This allows me to install my dotfiles on a new computer and use Vim without a lot of errors being generated because plugins aren't installed yet.
- Everything under [after](https://github.com/tweekmonster/dootfiles/tree/master/vim/after) is for settings that only matter to certain file types.

**Comment by gatspy** [comment]
yeah, i just stay at the level of use. your  way is suitable  for study and research, is too esoteric.
